contracts/NumberManager.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+contract NumberManager {
+
+    uint private totalSum;
+    uint public lastAddedNumber;
+
+    constructor(uint initialTotalSum) {
+        totalSum = initialTotalSum;
+        lastAddedNumber = 0;
+    }
+
+    function addNumber(uint number) public {
+        incrementTotalSum(number);
+        lastAddedNumber = number;
```

Comment:
use function setLastAddedNumber at here to set lastAddedNumber

Reply:
updated! Thanks for the suggestion